mapactionpy_arcmap/MapChef.py

```diff
@@ -56,25 +58,36 @@ def cook(self, productName, countryName):
             properties = self.layerProperties.get(layer)
             # Add layer to the report for later
             mapResult = MapResult(layer)
-            if (properties is not None):             
+            if (properties is not None):
                 layerFilePath = os.path.join(self.layerDirectory, (properties.layerName + ".lyr"))
                 if (os.path.exists(layerFilePath)):
                     self.dataFrame = arcpy.mapping.ListDataFrames(self.mxd, properties.mapFrame)[0]
                     layerToAdd = arcpy.mapping.Layer(layerFilePath)
-                    dataFilePath= os.path.join(self.root, "GIS", "2_Active_Data", properties.sourceFolder)
+                    # APS 06/09/2019 Please remove hardcoded path. See github comment
+                    dataFilePath = os.path.join(self.root, "GIS", "2_Active_Data", properties.sourceFolder)
```

Comment:
Please remove hardcoded path. Suggest you use something like:
```
cmf = CrashMoveFolder(path_to_cmf_description)
dataFilePath = cmf.active_data
```
See https://github.com/mapaction/mapactionpy_controller/blob/master/mapactionpy_controller/crash_move_folder.py